### PR TITLE
Added one line in hi.txt

### PR DESCRIPTION
**Pull request by amarasimhareddy03**

### CODE REVIEW

**Review by syed-salman-technoforte — CHANGES_REQUESTED** [review]
REMOVE 1st line "this is first line of hi.txt" to hi.txt

File: hi.txt [hunk 1]
```diff
@@ -1 +1,2 @@
-dcdnddndd
+## Removed line 1
```
Comment (syed-salman-technoforte) [inline hi.txt:1]: @amarasimhareddy03 please removed "##" from this line

**Comment by syed-salman-technoforte** [comment]
https://github.com/syed-salman-technoforte/test/blob/9ae7cdb5d14afdb8aec10ad4eb6e7c727bcbe818/hi.txt#L4-L6

@amarasimhareddy03  please change this lines to x y x

**Comment by syed-salman-technoforte** [comment]
@amarasimhareddy03, I have requested you xyz not for double aa bb cc. Please to the change 

https://github.com/syed-salman-technoforte/test/blob/e2221c0744d989ede51872da334739eef7fd308a/hi.txt#L4-L6

**Comment by amarasimhareddy03** [comment]
These changes i have done.
https://github.com/amarasimhareddy03/test/blob/86fec11152f53eb2a85482bc4075db94a4c511a2/hi.txt#L4-L6

**Comment by syed-salman-technoforte** [comment]
https://github.com/syed-salman-technoforte/test/blob/86fec11152f53eb2a85482bc4075db94a4c511a2/hi.txt#L4-L6

**Comment by amarasimhareddy03** [comment]
https://github.com/amarasimhareddy03/test/blob/86fec11152f53eb2a85482bc4075db94a4c511a2/hi.txt#L4-L6

**Comment by amarasimhareddy03** [comment]
https://github.com/syed-salman-technoforte/test/blob/86fec11152f53eb2a85482bc4075db94a4c511a2/hi.txt#L4-L6

**Comment by syed-salman-technoforte** [comment]
ok @amarasimhareddy03, i'm satisfied with the PR merging

**Pull request merged**